test/fully-connected.cc

```diff
@@ -759,8 +759,8 @@ TEST_F(FullyConnectedTestQC8, matches_operator_api)
 
   uint32_t bias_id = XNN_INVALID_VALUE_ID;
   ASSERT_EQ(
-    xnn_status_success, xnn_define_quantized_tensor_value(
-                          subgraph, xnn_datatype_qint32, 0, kernel_scale, bias_dims.size(), bias_dims.data(),
```

Comment:
The variable `kernel_scale` is no longer used and should be removed.

Reply:
good catch, thanks! Updated.